pkg/ast/spl/spl.peg

```diff
@@ -3295,10 +3433,102 @@ EarliestOnly <- CMD_EARLIEST EQUAL earliestTime:AbsoluteTimestamp !. {
     return node, nil
 }
 
-AbsoluteTimestamp <- [0-9][0-9] '/' [0-9][0-9] '/' [0-9][0-9][0-9][0-9] ':' [0-9][0-9] ':' [0-9][0-9] ':' [0-9][0-9] {
+RelIntegerAsString <- [-+][0-9]* {
+    return string(c.text), nil
+}
+
+WeekSnap <- ('w'[0-7]) {
```

Comment:
Just double checking: does splunk allow 0-7 for the weekday, or is it 0-6 or 1-7?

Reply:
w0 and w7 would be treated same. I just followed what they have.